src/main/kotlin/org/jetbrains/research/testspark/data/Technique.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.research.testspark.data
+
+/**
+ * Enum representing different techniques.
+ */
+enum class Technique {
```

Comment:
Why do we need a dedicated enum only for collectors?

Reply:
discussed in the slack

Reply:
What does `EA` stand for? `EA = EvoSuite Algorithm`? Better to write a full name.